CRT.c

```diff
@@ -94,21 +95,25 @@ const char* const* CRT_treeStr = CRT_treeStrAscii;
 
 static const Settings* CRT_settings;
 
-const char* CRT_degreeSign;
+#ifdef HAVE_LIBNCURSESW
+char CRT_degreeSign[MB_LEN_MAX] = "\xc2\xb0";
```

Comment:
Oops. Off-by-one error. The array size should be `MB_LEN_MAX  + 1`. Will fix this in the next rebase.